tests/test_ffi_c_interpreter.nit

```diff
@@ -15,7 +15,7 @@
 import core::kernel
 
 fun foo `{ printf("FOO\n"); `}
-fun bar(i: Int) `{ printf("BAR %ld\n", i); `}
+fun bar(i: Int) `{ printf("BAR %ld\n", (long)i); `}
```

Comment:
In this case, shouldn't the `%ld` part be changed instead of the cast to `long`?